src/js_types.hpp

```diff
@@ -461,7 +461,7 @@ struct Object {
     static typename ClassType::Internal* get_internal(ContextType ctx, const ObjectType&);
 
     template <typename ClassType>
-    static void set_internal(ContextType ctx, const ObjectType&, typename ClassType::Internal*);
+    static void set_internal(ContextType ctx, ObjectType&, typename ClassType::Internal*);
```

Comment:
What did removing `const` achieve?

Reply:
This makes the code able to mutate the "target" object when setting the internal, something we didn't have to do from "outside" of the engine specific code before. I need when setting the internal on JSC, but as this type is shared across all engines I need to update it here as well.